Sources/SWBTestSupport/BuildOperationTester.swift

```diff
@@ -713,6 +713,21 @@ package final class BuildOperationTester {
 
         }
 
+        package func checkNoTaskWithBacktraces(_ conditions: TaskCondition..., sourceLocation: SourceLocation = #_sourceLocation) {
+            for matchedTask in findMatchingTasks([]) {
+                Issue.record("found unexpected task matching conditions '\(conditions)', found: \(matchedTask)", sourceLocation: sourceLocation)
+                let backtraces = events.compactMap ({ (event) -> String? in
```

Comment:
This will show the first backtrace frame instead of following the full chain, but that might be enough in a lot of cases

Reply:
Ah, good point. I went ahead and just did it properly; we already had some backtrace enumeration code in other helpers already.